<?xml version="1.0" ?>
<launch>

    <arg name="config_name" default="keyboard_teleop" />

    <!--     Start the controller node -->
    <node name="keyboard_teleop_node" pkg="keyboard_teleop" type="keyboard_teleop_node.py" respawn="false" output="screen">
        <rosparam command="load" file="$(find keyboard_teleop)/config/$(arg config_name).yaml"/>
    </node>

</launch>
